tests/etc/test_devices.yml.example

```diff
@@ -178,18 +178,6 @@ keymile_nos:
   username: TEST
   password: TEST
 
-dlink_ds:
-  device_type: dlink_ds
-  ip: 192.168.50.10
-  username: admin
-  password: admin
-
-dlink_ds_telnet:
-  device_type: dlink_ds_telnet
-  ip: 192.168.50.10
-  username: admin
-  password: admin
-
```

Comment:
Rebase to fix.

Reply:
Hey apologies, I'm still getting used to Git and forking etc.. might take me a few tries to get it back to normal 😆